l10n_ch_payment_slip/invoice.py

```diff
@@ -67,15 +66,15 @@ def _is_generate_bvr(self, cr, uid, invoice, context=None):
         return (invoice.partner_bank_id and
                 invoice.partner_bank_id.state == 'bvr')
 
-    def _compute_bvr_ref(self, cr, uid, move_line, context=None):
+    def _compute_bvr_ref(self, cr, uid, move_line,
+                         sequence_int=0, context=None):
         ''' Default BVR reference computation '''
         res = ''
         if move_line.invoice.partner_bank_id.bvr_adherent_num:
             res = move_line.invoice.partner_bank_id.bvr_adherent_num
-        move_number = ''
-
+        move_number = str(sequence_int)
         if move_line.invoice.number:
-            compound = str(move_line.invoice.number) + str(move_line.id)
+            compound = str(move_line.invoice.number) + str(sequence_int)
```

Comment:
Can you explain why it is better to use this `sequence_int` instead of `move_line.id`, in a comment maybe? I don't understand it.